system/alternatives.py

```diff
@@ -123,10 +142,17 @@ def main():
                 if not link:
                     module.fail_json(msg="Needed to install the alternative, but unable to do so as we are missing the link")
 
-                module.run_command(
-                    [UPDATE_ALTERNATIVES, '--install', link, name, path, str(DEFAULT_LINK_PRIORITY)],
-                    check_rc=True
-                )
+                if need_to_force:
```

Comment:
@wislta-github You can (and should) use the same PR. You can just push new commits to your branch and they'll show up here. The preferred way (so you don't pollute the commit history) is to amend your commit with `git commit --amend` and then force-push to your branch with `git push -f <remote> alternatives-force`.